clients/drcachesim/CMakeLists.txt

```diff
@@ -720,20 +720,31 @@ ${debugopt}\n")
 endfunction ()
 
 if (X64)
-  set(CONFIG_INSTALL ${PROJECT_BINARY_DIR}/drcachesim.drrun64)
-  set(CONFIG_BUILD ${PROJECT_BINARY_DIR}/tools/drcachesim.drrun64)
+  set(CONFIG_INSTALL ${PROJECT_BINARY_DIR}/drmemtrace.drrun64)
+  set(CONFIG_BUILD ${PROJECT_BINARY_DIR}/tools/drmemtrace.drrun64)
+  # Support the older launcher name.
```

Comment:
Would be nice to xref i#6660 here and elsewhere.

Reply:
Done.